File: BaekJoon Online Judge/Realization(구현)/#2566 최댓값/최댓값.java
import java.util.*;

public class Main
{
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int max = Integer.MIN_VALUE;
		int x = 0, y = 0;
		
		for(int i = 0; i < 81; i++){
		    int num = sc.nextInt();
		    if(max < num){
		        max = num; x = i / 9; y = i % 9;
		    }
		}
		System.out.println(max + "\n" + (x+1) + " " + (y+1));
	}
}

File: BaekJoon Online Judge/Realization(구현)/#2566 최댓값/최댓값.kt
import java.io.StreamTokenizer

fun main() = StreamTokenizer(System.`in`.bufferedReader()).run {
    var max = Int.MIN_VALUE; var x = 0; var y = 0;
    repeat(81){
        nextToken()
        val num = nval.toInt()
        if(max < num){ max = num; x = it / 9; y = it % 9; }
    }
    println("$max\n${x+1} ${y+1}")
}